dev-proxy/PluginLoader.cs

```diff
@@ -45,8 +45,11 @@ public PluginLoaderResult LoadPlugins(IPluginEvents pluginEvents, IProxyContext
                 Assembly assembly = pluginLoadContext.LoadFromAssemblyName(new AssemblyName(Path.GetFileNameWithoutExtension(pluginLocation)));
                 IEnumerable<UrlToWatch>? pluginUrlsList = h.UrlsToWatch?.Select(ConvertToRegex);
                 ISet<UrlToWatch>? pluginUrls = null;
-                if (pluginUrlsList is not null)
-                {
+                if (pluginUrlsList is null || !pluginUrlsList.Any()){
+                    pluginUrlsList = globalUrlsToWatch.ToList();
+                }
+
+                if (pluginUrlsList.Any()){
```

Comment:
Since we've checked the pluginUrlList already above, why checking it again here? When would this check be falsy?

Also, let's drop the { to the next line for consistency.